common/client.go

```diff
@@ -177,6 +181,10 @@ func (c *DatabricksClient) configureHTTPCLient() {
 	if c.TimeoutSeconds == 0 {
 		c.TimeoutSeconds = 60
 	}
+	if c.RateLimit == 0 {
+		c.RateLimit = 1200
```

Comment:
maybe it makes sense to declare a constant somewhere, so it will be used in the provider conf as well?

Reply:
good idea